docker/build-ubuntu.ps1

```diff
@@ -37,18 +81,67 @@ function BuildUbuntuDockerImage ($version, $arch) {
     if ($(Test-Path $artifactsPackagePath) -eq $False) {
         throw "Package file does not exist."
     }
-    
+
     $dockerPackagePath = Join-Path -Path $DockerfileDir -ChildPath "RavenDB.tar.bz2"
     Copy-Item -Path $artifactsPackagePath -Destination $dockerPackagePath -Force
     Copy-Item -Path $RavenDockerSettingsPath -Destination $(Join-Path -Path $DockerfileDir -ChildPath "settings.json") -Force
-
     write-host "Build docker image: $version"
     $tags = GetUbuntuImageTags $repo $version $arch
     write-host "Tags: $tags"
 
     $fullNameTag = $tags[0]
 
-    docker build $DockerfileDir -f "$($DockerfileDir)/Dockerfile.$($arch)" -t "$fullNameTag"
+    if ([string]::IsNullOrEmpty($DebPackagePath)) {
+        $ubuntuVersion = GetUbuntuVersionFromDockerfile $DockerfileDir "Dockerfile.$arch"
+        SetupDebBuildEnvironment $arch $ubuntuVersion
+        
+        $archNameToMatch = switch ($arch) {
+            "x64" { "amd64"; break }
+            "arm32v7" { "arm32"; break }
+            "arm64v8" { "arm64"; break }
+            Default {
+                throw "ERROR: Unsupported architecture $($arch)"
+                exit 1
+            }
+        }
+        
+
+        if (!$NoCache) {
+            $matchingFile = Get-ChildItem $DockerfileDir | Where-Object { $_.Name -like "ravendb*$archNameToMatch*.deb" }
```

Comment:
Please check if it matches the new format https://github.com/ravendb/ravendb/pull/15942/commits/8662b5d33f8630e142304bcfe25c17b93e8bd975#diff-cc6d60c33f93b8880b349796c3f9e1221163ba37200bbef586b8dd79168fc0cdR85

Also please run with SetupMode Initial and check if it can obtain the cert - it has worked in our previous images.